app/scripts/services/specofferDictionaryService.js

```diff
@@ -2,94 +2,118 @@
 
 angular.module('admissionSystemApp')
 
-  .config(['$httpProvider', function($httpProvider) {
-    $httpProvider.defaults.headers.get = { 'Authorization' : 'Basic YWRtaW46bmltZGE=' };
-  }])
+.config(['$httpProvider', function($httpProvider) {
+  $httpProvider.defaults.headers.get = {
+    'Authorization': 'Basic YWRtaW46bmltZGE='
+  };
+}])
 
-  .factory('SpecofferDictionaryService', ['$http', '$q', function ($http, $q) {
+.factory('SpecofferDictionaryService', ['$http', '$q', function($http, $q) {
 
-    function requestConfig(item, limit, offset, customParams) {
-        var normalParams = {
-          limit: limit,
-          offset: offset
-        };
-        angular.extend(normalParams, customParams);
+  function requestConfig(item, limit, offset, customParams) {
+    var normalParams = {
+      limit: limit,
+      offset: offset
+    };
+    angular.extend(normalParams, customParams);
 
-        return {
-          method: 'GET',
-          url: 'http://176.36.11.25/api-lnu/' + item,
-          params: normalParams
-          // ,cache: true
-        };
-      }
+    return {
+      method: 'GET',
+      url: 'http://176.36.11.25/api-lnu/' + item,
```

Comment:
We should extract the server address to service and use it instead of copy from service to service